.formatter.exs

```diff
@@ -76,7 +76,17 @@ locals_without_parens = [
   on_mount: 1,
   slot: 1,
   slot: 2,
-  slot: 3
+  slot: 3,
+
+  # Phoenix.LiveViewTest
+  assert_patch: 2,
+  assert_patch: 3,
+  assert_patched: :2,
+  assert_push_event: 4,
+  assert_redirect: 2,
+  assert_redirect: 3,
+  assert_redirected: 2,
+  assert_reply: 3
```

Comment:
We already have `assert_reply/3` in line 53. I don't know if duplicates are a problem though.

Reply:
As for duplicates, they shouldn't affect the code, as the function that checks them will work the same (it basically just `Enum.any?` this list here: https://github.com/elixir-lang/elixir/blob/44107f1010229f58d899177e34eb04357a075ab3/lib/elixir/lib/code/formatter.ex#L180-L185

IMO we should leave it here, just in case someday someone removes one of these sections.